Modify S1116: Migrate to LayC (#3320)
SonarSource/rspec: rules/S1116/description.adoc

> Empty statements represented by a semicolon ``++;++`` are statements that do not perform any operation. They are often the result of a typo or a misunderstanding of the language syntax. For these reasons, it's a good practice to remove empty statements. They do not add any value and can only lead to confusion and errors.

Reviewer: Small fix and rephrasing:
```suggestion
Empty statements represented by a semicolon `;` are statements that do not perform any operation. They are often the result of a typo or a misunderstanding of the language syntax. 
It is a good practice to remove empty statements since they don't add value and lead to confusion and errors.
```

Author: Thanks 👍
